include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_reduce.h

```diff
@@ -426,96 +424,76 @@ __parallel_transform_reduce(oneapi::dpl::__internal::__device_backend_tag __back
 {
     auto __n = oneapi::dpl::__ranges::__get_first_range_size(__rngs...);
     assert(__n > 0);
+    using _Size = decltype(__n);
+
+    // Empirically found tuning parameters for typical devices.
+    constexpr _Size __max_iters_per_work_item = 32;
+    constexpr ::std::size_t __max_work_group_size = 256;
```

Comment:
Probably, is better to make request to a device, like `oneapi::dpl::__internal::__max_work_group_size(...)` and `oneapi::dpl::__internal::__max_sub_group_size(...)` ?